Bottom: eb9ff08acaa146e22c061adecd4964b457f2ae20
Top:    eb9ff08acaa146e22c061adecd4964b457f2ae20
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 04:56:01 +0100

Add configure checks for GtkGLEext, add to CFLAGS and LIBS

*** TODO: Need separate tests for libgl and libglut etc.. ***















---


